includes/class-assets.php

```diff
@@ -250,13 +250,17 @@ public function enqueue_css_override() {
         }
     }
 
-	public function enqueue_admin_scripts( $force = false ) {
+    /**
+     * Load resources on admin page
+     *
+     * @param string $hook The current admin page.
+     */
+	public function enqueue_admin_scripts( $hook ) {
```

Comment:
This is a public function, so we can't remove the $force parameter without digging thoroughly for what issues it may cause.

Reply:
It may need to change, as the button styling was also overriding those in other wp-admin pages. Ideally the assets should only load in BD admin pages. I will look for a workaround for the `force` param

Reply:
Sounds like the classes in the css need to change if are overriding buttons that aren't ours.